third_party/terraform/tests/resource_compute_forwarding_rule_test.go.erb

```diff
@@ -182,3 +182,58 @@ resource "google_compute_forwarding_rule" "foobar" {
 }
 `, poolName, ruleName)
 }
+
+func TestAccComputeForwardingRule_mirror(t *testing.T) {
```

Comment:
Because this test is only important if we are in the beta provider we can wrap the whole thing in the `<% unless version == 'ga' -%>` tags rather than just the `is_mirroring_collector = true`.

Reply:
Done, thanks.

Reply:
You'll need to move the `<% unless version ...` tag above the definition of `func TestAccComputeForwardingRule_mirror` or else the test will be rendered as GA but the function that defines the config won't, causing an error.

You can see the issue here: https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-3043-old..auto-pr-3043

Reply:
That makes sense, non-beta provider is no longer appearing in diffs